build.gradle

```diff
@@ -14,7 +14,7 @@ ext {
 group = "com.amazon.opendistroforelasticsearch"
 // Increment the final digit when there's a new plugin versions for the same opendistro version
 // Reset the final digit to 0 when upgrading to a new opendistro version
-version = "${opendistroVersion}.0" + (isSnapshot ? "-SNAPSHOT" : "")
+version = "${opendistroVersion}.1" + (isSnapshot ? "-SNAPSHOT" : "")
```

Comment:
please add a comment for "future generations" if/when we release the next minor version zip for ODFE